Xendit/Xendit.swift

```diff
@@ -43,6 +43,13 @@ import Foundation
                 return
             }
         }
+        
+        if cardData.cardCvn != nil && cardData.cardCvn != "" {
```

Comment:
This is redundant right? `cardData.cardCvn != nil`. Since guard below handles it. and `cardData.cardCvn != ""` probabaly isnt required, because cvn length will just be 0. I think we don't need this if statement

Reply:
this `if` is actually important because CVN is not required for tokenization therefore if there is no input in CVN we're skipping the validation